### PR TITLE
add fixes for postgres on dev

### PR DESCRIPTION
**Pull request by realalexhomer**
This branch fixes issues that were happening when using postgres on dev. 

Camelcase for sessions was switched to snake case.

A bug where sequelize wasn't automatically grouping the skills / candidates join tables was fixed.

### CODE REVIEW

File: server/models/CandidateModel.js [hunk 1]
```diff
@@ -20,9 +20,19 @@ const listByType = (listType) => {
           'author',
           [Sql.fn('MAX', Sql.col('comments.created_at')), 'created_at']
         ]
+      },
+      {
+        model: SkillModel,
+        attributes: [
```
Comment (jshor) [inline server/models/CandidateModel.js]: do we need to select attributes on SkillModel? the only columns are `id` and `text`

File: migrations/4.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// Author: Alex
+/**
+ * Renames session columns to be snake case rather than camel case.
+ * This is required in order to run raw queries in postgres as it flattens
+ * all identifiers to lower case. (see migration 2)
+ */
+
+module.exports = {
```
Comment (jshor) [inline migrations/4.js]: +1 for making the snake_case consistent

**Comment by jshor** [comment]
other than the possibly redundant attribute selection on SkillModel, LGTM

File: server/models/CandidateModel.js (hunk 1 above)
Comment (realalexhomer) [inline server/models/CandidateModel.js]: This is also part of the fix for the postgres error. We are going to be able to throw both fixes out when we switch to mysql.

**Comment by realalexhomer** [comment]
Closing this for now because we are trying to switch to mysql

File: migrations/4.js (hunk 2 above)
Comment (realalexhomer) [inline migrations/4.js]: Might need to merge this change but not the other ones.

**Pull request closed without merging**